app/views/moderations/index.html.erb

```diff
@@ -13,6 +13,11 @@
         <h2 class="crayons-subtitle-1">
           <%= @tag ? "##{@tag.accessible_name}" : t("views.moderations.all") %>
         </h2>
+        <% if FeatureFlag.enabled?(:moderator_role) %>
```

Comment:
Display notice about unavailable Mod Actions in the ModCenter-loaded Mod Panel.

Reply:
From the updated plan ([mentioned here](https://github.com/forem/forem/pull/18182/files#r927976188)) it seems like the FeatureFlag should be kept, right? Just double checking 🙂

Reply:
@fdocr So the feature flag covers just the `super-moderator` role activation. But this message applies to `admin` and `trusted` users also. So the message needs to show for them as well.